devlib/module/cgroups.py

```diff
@@ -180,13 +211,19 @@ def tasks(self, cgroup):
         tasks = {}
         for task in entries:
```

Comment:
I would re-write this loop to split only once for each task at the beginning: 

```
tid_str, tname, tcmdline = task.split(',', 2)
tid = int(tid_str)
```

(also note the limit on the split to handle the case where cmdline itself contains commas).

Reply:
Done